.projenrc.ts

```diff
@@ -111,6 +111,11 @@ const toolVersionsComponent = ProjenComponentProject.fromParent(monorepo, {
 	name: 'projen.component.tool-versions',
 })
 
+const dirEnvComponent = ProjenComponentProject.fromParent(monorepo, {
+	name: 'projen.component.dir-env',
+	deps: ['vitest'],
+})
+
```

Comment:
```suggestion
	name: 'projen.component.dir-env',
})
new Vitest(dirEnvComponent)

```

Reply:
To clarify, this will:
 1.) Create a [Vitest workspaces project config](https://vitest.dev/guide/workspace.html#configuration) with the project name as tests name and test file matchers.
 2.) Add project config path to [Vitest workspace config](https://vitest.dev/guide/workspace.html#defining-a-workspace)
 3.) Add any needed test related dependencies.
 4.) Add `test` and `test:watch` commands to the test/test:watch tasks for project.